cli/src/cli/index.ts

```diff
@@ -149,14 +154,29 @@ export const runCli = async () => {
   cliResults.flags = program.opts();
 
   /** @internal Used for CI E2E tests. */
-  let CIMode = false;
   if (cliResults.flags.CI) {
-    CIMode = true;
     cliResults.packages = [];
     if (cliResults.flags.trpc) cliResults.packages.push("trpc");
     if (cliResults.flags.tailwind) cliResults.packages.push("tailwind");
     if (cliResults.flags.prisma) cliResults.packages.push("prisma");
+    if (cliResults.flags.drizzle) cliResults.packages.push("drizzle");
     if (cliResults.flags.nextAuth) cliResults.packages.push("nextAuth");
+
+    if (cliResults.flags.prisma && cliResults.flags.drizzle) {
+      console.warn(
+        "Incompatible combination Prisma + Drizzle. Falling back to Prisma only."
+      );
+      cliResults.flags.drizzle = false;
+      cliResults.packages = cliResults.packages.filter(
+        (pkg) => pkg !== "drizzle"
+      );
+    }
+
+    return cliResults;
+  }
```

Comment:
i think i'm in favour of exiting with a non-zero code. 2 (Invalid command line usage or invalid input) seems to make sense.

why do you think we shouldn't? does it relate to our own actions that make sure the CLI works in CI?

Reply:
> why do you think we shouldn't? does it relate to our own actions that make sure the CLI works in CI?

Yes, it would probably be more work to make the CI matrix work in that case, maybe we can have a check in a catch block or something that just gracefully exits if `process.env.CI` is true?

Reply:
yea sounds good